Code/GraphMol/Depictor/testDepictor.cpp

```diff
@@ -1468,6 +1469,22 @@ M  END
   }
 }
 
+
+void testUseRingTemplates() {
```

Comment:
We are, of course, gonna need a bunch more tests. :-)

Reply:
Ah ok -- I moved this test to catch_tests!

I added most of the tests in python because its a little easier to write (specifically for checking if a mol followed a template). Should I move some of these to C++ as well? Or, I could add a few more that use different structures than the python tests.